libs/agno/agno/agent/agent.py

```diff
@@ -3234,19 +3253,37 @@ def get_relevant_docs_from_knowledge(
                 retriever_kwargs: Dict[str, Any] = {}
                 if "agent" in sig.parameters:
                     retriever_kwargs = {"agent": self}
+                if "filters" in sig.parameters:
+                    retriever_kwargs["filters"] = filters
```

Comment:
This means `get_relevant_docs_from_knowledge` can get `filters` that are not used. Right?

That may be confusing for the user, or is it handled somewhere else (communicating the filters were invalid / not applicable / not available)

Reply:
Yeh we should keep track of the filters available on the knowledge base and just warn the user if irrelevant filters are provided.

Reply:
<img width="770" alt="image" src="https://github.com/user-attachments/assets/bf98142e-931f-409f-a3f7-23ddd659935a" />

added something like this!